src/Compilers/Core/Portable/Operations/Loops/ILoopStatement.cs

```diff
@@ -21,6 +21,10 @@ public interface ILoopStatement : IOperation
         /// Body of the loop.
         /// </summary>
         IOperation Body { get; }
+        /// <summary>
+        /// Declarations local to the loop.
+        /// </summary>
+        ImmutableArray<ILocalSymbol> Locals { get; }
```

Comment:
General question not blocking this PR: would it be useful to have an interface for all IOperation nodes that declare locals?

Reply:
Good idea, let me file an issue to track this.

Reply:
Filed https://github.com/dotnet/roslyn/issues/22019